### PR TITLE
Adjust ffmpeg configuration command to dynamically fetch CPU core count

### PR DESCRIPTION
**Pull request by nicklasfrahm**
This is a potential fix for #20.

### CODE REVIEW

**Comment by nicklasfrahm** [comment]
@jaredpetersen I've updated it. It should get the core count and CPU architecture only from very basic bash commands. I will test it on my 3B+ as soon as I am home and I will also check if I have other RPis lying around to test it. Currently, I've only validated these commands on my [Windows Subsystem Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) with Ubuntu 16.04.

**Comment by jaredpetersen** [comment]
@nicklasfrahm Were you able to confirm that this works on your raspberry pi 3B+ and some of your other pis? Otherwise, looks great to me.

**Comment by nicklasfrahm** [comment]
@jaredpetersen Specifying a specific arch for `arm`, either `armhf`, `armel` or `armv7l`, is unnecessary, as [this](https://github.com/FFmpeg/FFmpeg/blob/master/configure#L4726) shows. I have removed it and verified that the command for the CPU thread count works on the following devices:
- Raspberry Pi 3B+
- Raspberry Pi 3B

**Comment by nicklasfrahm** [comment]
Would you mind if I also throw in an editor config to prevent problems with whitespaces in PRs? Something like this:

```txt
# top-most .editorconfig file
root = true

[*]
end_of_line = lf
insert_final_newline = true
charset = utf-8
indent_style = space
indent_size = 2
trim_trailing_whitespace = true

[*.md]
trim_trailing_whitespace = false
```

File: README.md [hunk 1]
```diff
@@ -111,9 +111,9 @@ sudo apt-get install libomxil-bellagio-dev
 wget -O ffmpeg.tar.bz2 https://ffmpeg.org/releases/ffmpeg-snapshot-git.tar.bz2
 tar xvjf ffmpeg.tar.bz2
 cd ffmpeg
-sudo ./configure --arch=armel --target-os=linux --enable-gpl --enable-omx --enable-omx-rpi --enable-nonfree
+sudo ./configure --arch=arm --target-os=linux --enable-gpl --enable-omx --enable-omx-rpi --enable-nonfree
 ```
-2. If you're working with a Raspbery Pi 2 or 3, then run `sudo make -j4` to build FFmpeg. If you're working with a Raspberry Pi Zero, then run `sudo make`.
+2. Now, run `sudo make -j$(grep -c ^processor /proc/cpuinfo)` to build FFmpeg and get a cup of coffee or two. This will take a while.
```
Comment (jaredpetersen) [inline README.md]: Remove the "Now," and just stick with "Run". It's not necessary and avoids having to deal with the debate over whether or not it's grammatically correct to put a comma after "Now" at the start of a sentence.

File: README.md [hunk 2]
```diff
@@ -111,9 +111,9 @@ sudo apt-get install libomxil-bellagio-dev
 wget -O ffmpeg.tar.bz2 https://ffmpeg.org/releases/ffmpeg-snapshot-git.tar.bz2
 tar xvjf ffmpeg.tar.bz2
 cd ffmpeg
-sudo ./configure --arch=armel --target-os=linux --enable-gpl --enable-omx --enable-omx-rpi --enable-nonfree
+sudo ./configure --arch=arm --target-os=linux --enable-gpl --enable-omx --enable-omx-rpi --enable-nonfree
```
Comment (jaredpetersen) [inline README.md:114]: Nice job on digging into FFmpeg's configure command. Wouldn't have even thought to look there. 👍

**Review by jaredpetersen — CHANGES_REQUESTED** [review]
Glad that you were able to test this on a few raspberry pis. Take care of the "Now" feedback and I'll merge it right away.

**Comment by jaredpetersen** [comment]
@nicklasfrahm I'd rather not commit any editor config files since there's a million different configs out there. I think it would be better if there was a linter or something like prettier installed that ensured that none of that was committed anyways. Not for this pull request though.

EDIT: Opened issue #22 per your feedback.

File: README.md (hunk 1 above)
Comment (nicklasfrahm) [reply README.md]: Sorry, I am not a native English speaker, but I appreciate every correction. 👍

**Comment by nicklasfrahm** [comment]
@jaredpetersen All comments are fixed and I will have a look at #22.

**Review by jaredpetersen — APPROVED** [review]
Looks great 👍

File: README.md (hunk 1 above)
Comment (jaredpetersen) [reply README.md]: Don't worry about it! Your written English is excellent. The "issue" around whether or not to put a comma after the word "Now" at the start of a sentence is apparently debated and it seems that both approaches are correct. It's just not something you see very often, so removing the word entirely avoids having to deal with the problem at all.

**Pull request merged**